docs/index/index.rst

```diff
@@ -23,17 +23,17 @@ User Guide
 
 The GeoTools User Guide is available online:
 
-:user:`User Guide <>` ( `stable <https://docs.geotools.org/stable/userguide/>`__ | `maintenance <https://docs.geotools.org/maintenance/userguide/>`__ )
+`User Guide <https://docs.geotools.org/stable/userguide/>`__ ( `stable <https://docs.geotools.org/stable/userguide/>`__ | `maintenance <https://docs.geotools.org/maintenance/userguide/>`__ )
     Tutorials, module reference and examples illustrating the uses of GeoTools. Updated with
     build instructions and advanced tutorials and integration instructions.
     
     The user guide is available for `download <https://sourceforge.net/projects/geotools/files/>`__
     as a zip file.
        
-:user:`Tutorials <tutorial/>` ( `stable <https://docs.geotools.org/stable/userguide/tutorial/>`__ | `maintenance <https://docs.geotools.org/maintenance/userguide/tutorial/>`__ )
+`Tutorials <tutorial/https://docs.geotools.org/stable/userguide/>`__ ( `stable <https://docs.geotools.org/stable/userguide/tutorial/>`__ | `maintenance <https://docs.geotools.org/maintenance/userguide/tutorial/>`__ )
```

Comment:
``<tutorial/https://docs.geotools.org/stable/userguide/>`__``

This is looking strange too...